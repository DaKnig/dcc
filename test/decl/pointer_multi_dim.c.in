int restrict * const * volatile * p;
