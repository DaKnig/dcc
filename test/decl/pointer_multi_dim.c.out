tree representation of pointer_multi_dim.c.in:
p is a 
	pointer to:
			volatile pointer to:
					const pointer to:
			restrict i16 ;
